docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,229 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+## Pre-requisites
+
+To install the Helm chart the only requirement is [Helm](https://helm.sh/).
+However, additional executables may be required to create/manage the cluster(s) (_e.g._, Kind and kubectl),
+to join Workload Execution Clusters (WEKs) (_e.g._, clusteradm),
+and to interact with Control Planes (_e.g._, kubectl), _etc_.
+For such purpose, a full list of executable that may be required can be fund [here](./pre-reqs.md).
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+This can be:
+
+1. A local **Kind** cluster with an ingress with SSL passthrough and a mapping to host port 9443

```

Comment:
A lot of the information in this section is not specific to `kind`. It is equally true for k3s, and (I suspect) any other sort of cluster that is not OpenShift. Also, a lot of this information is not specific to the hub chart.

Does not have to be addressed in this PR.

Reply:
The manual step by step starts with a `kflex init -c`... making most of this information unnecessary

Reply:
`kflex init -c` is specific to `kind`. A lot of the information in this section is not specific to `kind`. Some of the information here is not specific to the hub chart, it is general considerations that are true independent of the Helm chart: the domain name and ingress port number of the hosting cluster are important configuration facts that have broader significance than just their appearance in this chart.

Reply:
thank you